.travis.yml

```diff
@@ -30,12 +30,17 @@ before_install:
   # Install symfony/error-handler on compatible PHP versions to avoid a deprecation warning of the old DebugClassLoader and ErrorHandler classes
   - if [[ "$TRAVIS_PHP_VERSION" != 5.* && "$TRAVIS_PHP_VERSION" != 7.0 && "$SYMFONY_VERSION" = "" ]]; then composer require --no-update --dev symfony/error-handler "^4.4 || ^5.0"; fi;
   - sh -c 'if [ "$SYMFONY_VERSION" != "" ]; then composer require -n --no-update symfony/lts=$SYMFONY_VERSION; fi;'
+  - if (( "${TRAVIS_PHP_VERSION:0:1}" >= "8" )) ; then composer require --dev --no-update  "phpunit/phpunit:^7.5.18"; fi;
 
 install:
-  - composer install -n --prefer-dist
+  - if (( "${TRAVIS_PHP_VERSION:3}" == "snapshot" )) ; then composer install --ignore-platform-req=php -n --prefer-dist; else  composer install  -n --prefer-dist; fi;
```

Comment:
Is this change needed now, when PHP 8.0 is officially released?

Reply:
@aik099 Yes cause of ```mink/driver-testsuite``` does not support PHP 8.0 and composer install would fail otherwise.